### PR TITLE
Assorted UI cleanup

### PR DESCRIPTION
**Pull request by cnunciato**
As part of the on-prem and GitHub-detachment work, this change takes care of some additional cleanup, including:

* Removing dead GitHub-related code
* Removing the footer component, which is no longer used
* Removing the feature flag for personal access tokens
* Removing unused configuration entries, and alphabetizing the rest
* Pointing the Tutorials and Download URLs to their updated locations
* Fixing an overlooked link on the sign-in view
* Making `enable_builder` false by default

Signed-off-by: Christian Nunciato <chris@nunciato.org>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-api-proxy/default.toml [hunk 1]
```diff
@@ -1,47 +1,35 @@
...
-app_url      = "https://github.com/apps/habitat-builder"
 app_id       = 5565
+app_url      = "https://github.com/apps/habitat-builder"
+url          = "https://api.github.com"
 
 [oauth]
-provider       = "github"
-client_id      = ""
 authorize_url  = "https://github.com/login/oauth/authorize"
+client_id      = ""
```
Comment (chefsalim) [inline components/builder-api-proxy/default.toml]: wierd that a bunch of lines are just getting re-ordered..

**Comment by chefsalim** [comment]
tenor-47646140

**Comment by chefsalim** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

File: components/builder-api-proxy/default.toml (hunk 1 above)
Comment (cnunciato) [reply components/builder-api-proxy/default.toml]: Yep -- I noted in the PR description that I reordered these to alphabetize them, just to make things easier to scan for.

Comment (chefsalim) [reply components/builder-api-proxy/default.toml]: 👍

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to pass the test suite](https://travis-ci.org/habitat-sh/builder/builds/356426438).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/builder/builds/356426438) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/builder/builds/356426438).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/builder/builds/356426438).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**